core/src/trezor/lvglui/i18n/locales/ru.py

```
@@ -357,8 +357,8 @@
     "Удалить",
     "Установка обоев",
     "Экран блокировки",
-    "Во время блокировки коснитесь экрана, чтобы разбудить дисплей.",
-    "Нажмите для пробуждения",
+    "Во время блокировки дважды коснитесь экрана, чтобы разбудить дисплей.",
```

Ruff (0.11.9)

String contains ambiguous `В` (CYRILLIC CAPITAL LETTER VE). Did you mean `B` (LATIN CAPITAL LETTER B)?

(RUF001)